docs-standards/20-style-the-content/09-tables.md

```diff
@@ -0,0 +1,39 @@
+# Tables
+
+Tables provide a simple way to summarize information. Use a table instead of [bullet lists](./06-bullet-lists.md) when you need to describe the elements on the list.
+
+- Bullet list example (incorrect):
+
+  There are four types of items:
+
+  - **Item 1** <br>Item 1 description
+  - **Item 2** <br>Item 2 description
+  - **Item 3** <br>Item 3 description
+  - **Item 4** <br>Item 4 description
+
+- Table format example (**correct**):
+
+    | Item   | Description          |
+    |:-------|:---------------------|
+    | Item 1 | Item 1 description |
+    | Item 2 | Item 2 description |
+    | Item 3 | Item 3 description |
+    | Item 4 | Item 4 description |
+
+You can also use tables to compare the available options across different elements.
+
+    | Features   | Desktop | Mobile | Web |
+    |:-----------|:--------|:-------|:----|
+    | Feature A  |   ✅    |   ✅   |     |
+    | Feature B  |   ✅    |        |     |
+    | Feature C  |         |   ✅   | ✅  |
+
+> ☝️ **Tip:**
+>
+> You can use links in tables except if the table is part of a procedure step.
+
+Use the following guidelines when using periods in a table for punctuation:
```

Comment:
Punctuation guidelines edited based on the discussion here: https://github.com/status-im/ift-docs/discussions/39